src/docfx/lib/git/GitUtility.cs

```diff
@@ -163,6 +163,14 @@ public static Task PruneWorkTrees(string cwd)
         public static Task<string> Revision(string cwd, string branch = "HEAD")
            => ExecuteQuery(cwd, $"rev-parse {branch}");
 
+        public static void CheckMergeConflictMarker(string content, string file)
+        {
+            if ((content.StartsWith("<<<<<<<") || content.Contains("\n<<<<<<<")) && content.Contains("\n>>>>>>>"))
```

Comment:
`"<<<<<<<"` => `"<<<<<<< "` and also check `=======`? to avoid false positive as much as possible